test/functional/rpc_getpeerinfo_deprecation.py

```diff
@@ -26,7 +26,7 @@ def test_banscore_deprecation(self):
 
     def test_addnode_deprecation(self):
         self.restart_node(1, ["-deprecatedrpc=getpeerinfo_addnode"])
-        connect_nodes(self.nodes[0], 1)
+        self.connect_nodes(0, 1)
```

Comment:
The scripted-diff script seems to have missed the import in this file.

Reply:
Thanks for catching this!